tool/lib/iso15765_2.py

```diff
@@ -69,15 +69,20 @@ def clear_filters(self):
         """Remove arbitration ID filters"""
         self._set_filters(None)
 
-    def send_message(self, data, arbitration_id):
+    def send_message(self, data, arbitration_id, force_extended=False):
         """
         Transmits a message using 'arbitration_id' and 'data' on 'self.bus'
 
         :param data: Data to send
         :param arbitration_id: Arbitration ID to use
         :return: None
         """
-        msg = can.Message(arbitration_id=arbitration_id, data=data)
+        if arbitration_id <= ARBITRATION_ID_MAX:
+            is_extended = force_extended or False
```

Comment:
What is the purpose of the `or False` part? From what I can tell, this will never affect the value of the boolean assignment.

If I am reading your intent correctly, the multi-line check and assignment could simply be expressed like this: 
`is_extended = force_extended or arbitration_id > ARBITRATION_ID_MAX`

Reply:
Looking at it now, the `or False` is there strictly for readability and can be removed.